sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1774,6 +1773,21 @@ object SQLConf {
     .booleanConf
     .createWithDefault(false)
 
+  object IntervalStyle extends Enumeration {
+    type IntervalStyle = Value
+    val SQL_STANDARD, ISO_8601, MULTI_UNITS = Value
+  }
+
+  val INTERVAL_STYLE = buildConf("spark.sql.intervalOutputStyle")
+    .doc("Display format for interval values. The value SQL_STANDARD will produce output" +
```

Comment:
display is not accurate as this controls the cast behavior as well. How about `When converting interval values to strings (e.f. for display), this config decides the interval string format. The value SQL_STANDARD  ...`

Reply:
OK, I will make it more clearer